tests/AssociatedJobInfoClientTestsData/GetAssociatedJobResponse_ForkPR.json

```diff
@@ -0,0 +1,488 @@
+{
```

Comment:
This test data should be pared down to what we actually care about. This is far more than is necessary for the tests to do their part, and make it's difficult to understand what the test is doing.

Reply:
Once it's pared down, I'd like to see it moved into the test itself.  It's really hard to reason about a test when I have to go read a bunch of external files to understand what it's doing.